homeassistant/components/whirlpool/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""The Whirlpool Sixth Sense integration."""
+import logging
+
+import aiohttp
+from whirlpool.auth import Auth
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import AUTH_INSTANCE_KEY, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Whirlpool Sixth Sense from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    auth = Auth(entry.data["username"], entry.data["password"])
+    try:
+        await auth.do_auth(store=False)
+    except aiohttp.ClientError as ex:
+        _LOGGER.error("Connection error")
```

Comment:
```suggestion
```

Reply:
This will flood the log with errors when the service is down or there is a network problem as it will log every time its retried.  Since you are passing the error to the exception we shouldn't log here.